internal/fsutils/fsutils_test.go

```diff
@@ -136,3 +136,47 @@ func TestFSUtils_StringMatchMask(t *testing.T) {
 		})
 	}
 }
+
+func TestFSUtils_PlainStrings_StringMatchMask(t *testing.T) {
+	type args struct {
+		str     string
+		pattern string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "test module-name:filename with exact name",
+			args: args{
+				str:     "managed-pg:/enabled",
+				pattern: "managed-pg:/enabled",
+			},
+			want: true,
+		},
+		{
+			name: "test module-name:filename with glob mask",
+			args: args{
+				str:     "managed-pg:/.venv/lib/python3.13/site-packages/deckhouse/__init__.py",
+				pattern: "managed-pg:/.venv/**/*",
+			},
+			want: true,
+		},
+		{
+			name: "plain string with mask",
+			args: args{
+				str:     "test-string.delimited.by.dots",
+				pattern: "*.by.dots",
```

Comment:
If I understand correctly, then the pattern will fit any module without restrictions?
Do we want to add regexp only for parsing the file path? Or for the whole line?
If only for the path, then it is reasonable, if for the string, then in my opinion, it will introduce unnecessary errors and unprovable behavior.